src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -1025,12 +1025,6 @@ private void testIsStudentsInSameTeam() {
 
         ______TS("students of different teams");
 
-        StudentAttributes student5InCourse1 = dataBundle.students.get("student5InCourse1");
```

Comment:
Why is this removed?

Reply:
Sorry about this. I removed thinking it is the same tests. Will add it back